### PR TITLE
Code Readability - Clothing

### PR DESCRIPTION
**Pull request by KalevTait**
## What Does This PR Do
* Converts 0/1 to FALSE/TRUE where those #defines make the code more readable. Note: There are a few cases where this converts 0.0/1.0 to FALSE/TRUE - this change is consistent with how the variable are used.
* Converrs -1/0/1/2 to FLASH_PROTECTION_SENSITIVE/FLASH_PROTECTION_NONE/FLASH_PROTECTION_FLASH/FLASH_PROTECTION_WELDER where those #defines make the code more readable.
* Removes some unnecessary brackets in if statements
* Removes some unnecessary == in if statements

## Why It's Good For The Game
More readable code is easier to work with.

### CODE REVIEW

File: code/modules/clothing/head/misc_special.dm [hunk 1]
```diff
@@ -115,7 +115,7 @@
 
 /obj/item/clothing/head/cakehat/attack_self(mob/user as mob)
 	if(status > 1)	return
-	src.onfire = !( src.onfire )
+	src.onfire = !src.onfire
 	if(src.onfire)
 		src.force = 3
 		src.damtype = "fire"
```
Comment (hal9000PR) [inline code/modules/clothing/head/misc_special.dm:121]: `src.` here is implied, so its not needed.

File: code/modules/clothing/head/misc_special.dm [hunk 2]
```diff
@@ -115,7 +115,7 @@
 
 /obj/item/clothing/head/cakehat/attack_self(mob/user as mob)
```
Comment (hal9000PR) [inline code/modules/clothing/head/misc_special.dm:116]:
```suggestion
/obj/item/clothing/head/cakehat/attack_self(mob/user)
```
while you are here.

File: code/modules/clothing/spacesuits/alien.dm [hunk 3]
```diff
@@ -228,7 +228,7 @@
 	if(src.magpulse)
```
Comment (hal9000PR) [inline code/modules/clothing/spacesuits/alien.dm:228]: implied `src`

File: code/modules/clothing/spacesuits/chronosuit.dm [hunk 4]
```diff
@@ -28,10 +28,10 @@
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	var/obj/item/clothing/head/helmet/space/chronos/helmet = null
 	var/obj/effect/chronos_cam/camera = null
-	var/activating = 0
-	var/activated = 0
+	var/activating = FALSE
+	var/activated = FALSE
 	var/cooldowntime = 50 //deciseconds
```
Comment (hal9000PR) [inline code/modules/clothing/spacesuits/chronosuit.dm:33]:
```suggestion
	var/cooldowntime = 5 SECONDS
```

File: code/modules/clothing/spacesuits/chronosuit.dm [hunk 5]
```diff
@@ -173,16 +173,16 @@
 				to_chat(user, "logout")
 		src.flags &= ~NODROP
```
Comment (hal9000PR) [inline code/modules/clothing/spacesuits/chronosuit.dm:174]: implied src

File: code/modules/clothing/spacesuits/chronosuit.dm [hunk 6]
```diff
@@ -144,17 +144,17 @@
...
 		cooldown = world.time + cooldowntime
-		activating = 0
+		activating = FALSE
 		return 0
 
 /obj/item/clothing/suit/space/chronos/proc/deactivate()
 	if(activated)
-		activating = 1
+		activating = TRUE
 		var/mob/living/carbon/human/user = src.loc
```
Comment (hal9000PR) [inline code/modules/clothing/spacesuits/chronosuit.dm:158]: implied src

**Review by hal9000PR — DISMISSED** [review]

File: code/modules/clothing/head/misc_special.dm [hunk 7]
```diff
@@ -113,18 +113,18 @@
...
 	if(status > 1)	return
-	src.onfire = !( src.onfire )
-	if(src.onfire)
-		src.force = 3
-		src.damtype = "fire"
-		src.icon_state = "cake1"
+	onfire = !onfire
+	if(onfire)
+		force = 3
+		damtype = "fire"
```
Comment (farie82) [inline code/modules/clothing/head/misc_special.dm:121]:
```suggestion
		damtype = BURN
```

File: code/modules/clothing/head/misc_special.dm [hunk 8]
```diff
@@ -113,18 +113,18 @@
...
+		icon_state = "cake1"
 		START_PROCESSING(SSobj, src)
 	else
-		src.force = null
-		src.damtype = "brute"
-		src.icon_state = "cake0"
+		force = null
+		damtype = "brute"
```
Comment (farie82) [inline code/modules/clothing/head/misc_special.dm:126]:
```suggestion
		damtype = BRUTE
```

File: code/modules/clothing/head/misc_special.dm [hunk 9]
```diff
@@ -113,18 +113,18 @@
...
+		icon_state = "cake0"
 	return
```
Comment (farie82) [inline code/modules/clothing/head/misc_special.dm:128]:
```suggestion
```

File: code/modules/clothing/head/misc_special.dm [hunk 10]
```diff
@@ -113,18 +113,18 @@
 	if(istype(location, /turf))
 		location.hotspot_expose(700, 1)
 
-/obj/item/clothing/head/cakehat/attack_self(mob/user as mob)
+/obj/item/clothing/head/cakehat/attack_self(mob/user)
 	if(status > 1)	return
```
Comment (farie82) [inline code/modules/clothing/head/misc_special.dm:117]:
```suggestion
	if(status > 1)
		return
```

**Review by farie82 — APPROVED** [review]

**Pull request merged**